examples/jupyter/expansion-filters.ipynb

```diff
@@ -437,6 +439,1116 @@
+    "settings.inactive = 20\n",
+    "settings.particles = 100000"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Defining the tally is relatively straightforward. One simply needs to list 'flux' as a score and then add an expansion filter. For this case, we will want to use the `SpatialLegendreFilter`, `ZernikeFilter`, `ZernikeRadialFilter` classes which multiplies tally scores by Legendre, azimuthal Zernike and  radial-only Zernike polynomials evaluated on normalized spatial positions along radial and axial directions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a flux tally\n",
+    "flux_tally_legendre = openmc.Tally()\n",
+    "flux_tally_legendre.scores = ['flux']\n",
+    "\n",
+    "# Create a Legendre polynomial expansion filter and add to tally\n",
+    "order = 10\n",
+    "cell_filter = openmc.CellFilter(fuel)\n",
+    "legendre_filter = openmc.SpatialLegendreFilter(order, 'z', zmin, zmax)\n",
+    "flux_tally_legendre.filters = [cell_filter, legendre_filter]\n",
+    "\n",
+    "# Create a Zernike azimuthal polynomial expansion filter and add to tally \n",
+    "flux_tally_zernike = openmc.Tally()\n",
+    "flux_tally_zernike.scores = ['flux']\n",
+    "zernike_filter          = openmc.ZernikeFilter(order=order, x=0.0, y=0.0, r=radius)\n",
+    "flux_tally_zernike.filters  = [cell_filter, zernike_filter]\n",
+    "\n",
+    "# Create a Zernike radial polynomial expansion filter and add to tally \n",
+    "flux_tally_zernike1d = openmc.Tally()\n",
+    "flux_tally_zernike1d.scores = ['flux']\n",
+    "zernike1d_filter          = openmc.ZernikeRadialFilter(order=order, x=0.0, y=0.0, r=radius)\n",
+    "flux_tally_zernike1d.filters  = [cell_filter, zernike1d_filter]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last thing we need to do is create a `Tallies` collection and export the entire model, which we'll do using the `Model` convenience class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tallies = openmc.Tallies([flux_tally_legendre, flux_tally_zernike, flux_tally_zernike1d])\n",
+    "model = openmc.model.Model(geometry=geom, settings=settings, tallies=tallies)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Running a simulation is now as simple as calling the `run()` method of `Model`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "PosixPath('/mnt/d/MIT_CRPG/openmc_develop/openmc/examples/jupyter/statepoint.100.h5')"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.run(output=False)"
```

Comment:
This function will now return the path to the statepoint file, which you can use when calling `openmc.StatePoint` rather than hard-coding the path 'statepoint.100.h5', i.e.,
```python
sp_file = model.run(output=False)

with openmc.StatePoint(sp_file) as sp:
   ...
```